lib/gds_api/test_helpers/account_api.rb

```diff
@@ -82,6 +82,28 @@ def stub_account_api_unauthorized_user_info(**options)
         )
       end
 
+      ########################
+      # DELETE /api/oidc-users
+      ########################
+
+      def stub_account_api_success_delete_account(subject_identifier:, **options)
+        stub_account_api_request(
+          :delete,
+          "/api/oidc-users/#{subject_identifier}",
+          response_status: 204,
+          **options,
+        )
+      end
+
+      def stub_account_api_not_found_delete_account(subject_identifier:, **options)
```

Comment:
And this should be `stub_account_api_delete_user_by_subject_identifier_does_not_exist`